02-setup.md

```diff
@@ -48,9 +49,17 @@ He also has to set his favorite text editor, following this table:
 | Notepad++ (Win)    | `$ git config --global core.editor "'c:/program files (x86)/Notepad++/notepad++.exe' -multiInst -notabbar -nosession -noPlugin"`|
 | Kate (Linux)       | `$ git config --global core.editor "kate"`       |
 | Gedit (Linux)      | `$ git config --global core.editor "gedit -s -w"`   |
+| Scratch (Linux)       | `$ git config --global core.editor "scratch-text-editor"`  |
 | emacs              | `$ git config --global core.editor "emacs"`   |
+
+It is also possible to configure "Vim" as the text editor for Git if you want to change it back. Note that Vim is the default editor for Git so this is only needed if you have changed it already.
```

Comment:
I prefer to leave the table as it was (not making `vim` a new row). Instead of mentioning that `vim` is the default, we could mention that you can change the configuration (the editor and all the rest) as many times as you want. This is a question that shows up in the workshops and some people is not used to be able to configure computer programs as they please (they tend to worry about picking the "wrong" editor and having to stuck with it for life).

but I agree that it's very, very important to include the